.travis.yml

```diff
@@ -58,11 +57,15 @@ jobs:
         - curl -sSL "https://github.com/karalabe/xgo/blob/647f256c447ee20f9bf13ebc42e612d55994a383/docker/base/patch.tar.xz?raw=true" | xz -dc - | tar -xf -
         - mv v1 "${OSXCROSS_PATH}/SDK/MacOSX${SDK_VERSION}.sdk/usr/include/c++/v1"
         - rm -rf /tmp/osxcross "${OSXCROSS_PATH}/SDK/MacOSX${SDK_VERSION}.sdk/usr/share/man"
-      script:
         - cd $TRAVIS_BUILD_DIR
+      script:
         - ./sbt assembly || travis_terminate 1
         - ./sbt publishLocal
         - openssl aes-256-cbc -K $encrypted_97aef7f4ae04_key -iv $encrypted_97aef7f4ae04_iv -in key.asc.enc -out key.asc -d
         - gpg --no-default-keyring --primary-keyring ./project/.gnupg/pubring.gpg --secret-keyring ./project/.gnupg/secring.gpg --keyring ./project/.gnupg/pubring.gpg --fingerprint --import key.asc
         - ./sbt publishSigned
         - ./sbt sonatypeRelease
+    - <<: *release
+      name: 'Check that project can be cross-compiled correctly'
+      stage: test
+      script: ./sbt assembly test
```

Comment:
```suggestion
      script:
        - ./sbt assembly test
        - cd $TRAVIS_BUILD_DIR
```

Reply:
I have included `cd $TRAVIS_BUILD_DIR` at the end of `release install`, sinc it also was needed for `release script`. Now we have reproduced the same error as in the release. I am going to try and fix the MacOs sdk problem. Just to clarify, did you have to package manually the sdk and upload it to `https://s3.dockerproject.org/darwin/v2/MacOSX10.11.sdk.tar.xz`?

Reply:
I have the 10.13 version here https://github.com/phracker/MacOSX-SDKs/releases

Reply:
> I have included cd $TRAVIS_BUILD_DIR at the end of release install

Is that somewhere in this PR?